addons/binding/org.openhab.binding.freeboxV5/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you forgot to remove some files here

Reply:
I missed something during a git command. Fixed.